contracts/tasks/execute-payouts.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * @notice Given a data file with contribution data and merkle tree data, this script automates
+ * the match payout flow of:
+ *   1. Deploying the Merkle Distributor based on the payout data's merkle root
+ *   2. Transferring the match funds to the Merkle Distributor
+ *   3. Calling the bulk payout function on the distributor
+ *
+ * To use this, create a file in the `payout-data` folder called <dataName>.data.ts, and pass
+ * the name of the data set as an argument when calling this script. See sample usage for the
+ * dGrants POC round below.
+ *
+ * @dev Sample usages
+ * WARNING: Do not use a network other than hardhat until you are ready to send real transactions
+ *
+ * NOTE: Currently you'll need to replace `19443600` with `20913000` in hardhat.config.ts to ensure
```

Comment:
why do we not set it to `20913000` directly ?

Reply:
Yea, I probably should make that change. I didn't because it breaks gas snapshots, but those can be updated so I'll change this

Reply:
Fixed in a91b482